### DISCUSSION TITLE
Cloudlog down?

### DISCUSSION THREAD

**Discussion by soakland5** [magicbug/Cloudlog#2027] [Q&A]
Is it just me or is Cloudlog down?

**Comment by magicbug** [comment]
Hosted instances are all up and running.

**Comment by soakland5** [comment]
Can you check mine out ? W1AL

**Reply by magicbug** [reply]
Can confirm your account is running correctly

**Comment by soakland5** [comment]
Yes I see it now. Must have been my VPN. Thanks Peter
